justfile

```diff
@@ -0,0 +1,597 @@
+
+# Default recipe - show help
+default:
+    @just --list
+
+# =============================================================================
+# Backend
+# =============================================================================
+
+# Run any backend command (e.g., just b init, just b test)
+[doc("Run backend command: just b <cmd>")]
+backend *args:
+    @just --justfile backend/justfile --working-directory backend {{ args }}
+
+# Shortcut alias for backend
+alias b := backend
+
+# =============================================================================
+# Web-Frontend
+# =============================================================================
+
+# Run any web-frontend command (e.g., just f lint, just f test)
+[doc("Run frontend command: just f <cmd>")]
+frontend *args:
+    @just --justfile web-frontend/justfile --working-directory web-frontend {{ args }}
+
+# Shortcut alias for frontend
+alias f := frontend
+
+# =============================================================================
+# Docker Compose
+# =============================================================================
+
+_dc_help:
+    @echo "Usage: just dc <cmd> [args]       (production - uses published images)"
+    @echo "       just dc-dev <cmd> [args]   (development - builds dev images)"
+    @echo "       just dc-prod <cmd> [args]  (production images built locally)"
+    @echo ""
+    @echo "Examples:"
+    @echo "  just dc-dev build --parallel     # Build all dev images"
+    @echo "  just dc-dev build backend        # Build specific service"
+    @echo "  just dc-dev up -d                # Start containers (detached)"
+    @echo "  just dc-dev up -d backend db     # Start specific services"
+    @echo "  just dc-dev stop                 # Stop containers (keep volumes)"
+    @echo "  just dc-dev down                 # Stop and remove containers"
+    @echo "  just dc-dev logs -f backend      # Follow logs for a service"
+    @echo "  just dc-dev exec backend bash    # Open shell in container"
+    @echo "  just dc-dev ps                   # Show running containers"
+    @echo ""
+    @echo "Optional services (storybook, flower) - not started by default:"
+    @echo "  just dc-dev --profile optional up -d    # Include optional services"
+    @echo "  just dc-dev-full up -d                  # Shorthand for above"
+    @echo ""
+    @echo "Production (uses published images from registry):"
+    @echo "  just dc up -d"
+    @echo "  just dc pull"
+    @echo ""
+    @echo "Build production images locally:"
+    @echo "  just dc-prod build --parallel"
+
+# Production compose (base docker-compose.yml only)
+[doc("Docker compose (prod): just dc <cmd> [args]")]
+dc *ARGS:
+    #!/usr/bin/env bash
+    if [ -z "{{ ARGS }}" ]; then
+        just _dc_help
+    else
+        docker compose -f docker-compose.yml {{ ARGS }}
+    fi
+
+# Dev compose (includes docker-compose.dev.yml overlay)
+[doc("Docker compose (dev): just dc-dev <cmd> [args]")]
+dc-dev *ARGS:
```

Comment:
Before, with `./dev.sh`, I was able to run the `./dev.sh` command, it started the dev containers and then automatically opened tabs and attached into it. This was quite convenient for me. What do you think of introducing a similar command now? It can be a separate command that just attaches onto the container. Ideally something that works better with Mac this time because with ./dev.sh you would had to keep focus on the terminal.